Heading/tests/Heading-specs.js

```diff
@@ -1,33 +1,50 @@
-import {shallow} from 'enzyme';
+import {render} from '@testing-library/react';
 
 import {HeadingBase} from '../Heading';
-import css from '../Heading.module.less';
 
 describe('Heading Specs', () => {
 
 	test('should render a Heading with content', () => {
 		const content = 'Hello Heading!';
+		const {getByTestId} = render(<HeadingBase data-testid="heading">{content}</HeadingBase>);
+		const heading = getByTestId('heading').textContent;
 
-		const subject = shallow(
-			<HeadingBase>{content}</HeadingBase>
-		);
+		expect(heading).toBe(content);
+	});
+
+	test('should add the showLine class', () => {
+		const {getByTestId} = render(<HeadingBase data-testid="heading" showLine>check</HeadingBase>);
+		const heading = getByTestId('heading');
 
-		const expected = content;
-		const actual = subject.text();
+		const expected = 'showLine';
 
-		expect(actual).toBe(expected);
+		expect(heading).toHaveClass(expected);
```

Comment:
```suggestion
		expect(heading.className).toContain(expected);
```

Reply:
You have to replace that line in all following tests otherwise the tests will fail.

Reply:
I forgot to import '@testing-library/jest-dom'. Should be ok now